nemo_skills/code_execution/local_sandbox/local_sandbox_server.py

```diff
@@ -20,14 +20,26 @@
 import subprocess
 import sys
 import tempfile
+import signal
 from io import StringIO
 
 from flask import Flask, request
 
 app = Flask(__name__)
 
+MEM_LIMIT_BYTES = 10 * 1024 ** 3          # 10 GiB
```

Comment:
Can you accept this from `NEMO_SKILLS_SANDBOX_MEM_LIMIT` env var if it's provided? We had a request to let people customize this as there is probably not a single limit that works for any use-cases

Reply:
Yes but I think that would be better suited for the `SANDBOX_MEMORY_LIMIT` var which is passed to docker run. Let me know what you think.